src/testIntegration/java/uk/gov/justice/digital/delius/controller/secure/ReferenceDataResourceTest.java

```diff
@@ -1,217 +0,0 @@
-package uk.gov.justice.digital.delius.controller.secure;
```

Comment:
Should have been deleted in earlier PR, these tests had already been merged into ReferenceDataAPITest